examples/npc_only.py

```diff
@@ -26,18 +26,18 @@
     location=args.location,
     agent_count=10,
     monitor_infractions=True,
-    render_birdview=True,
     ego_agent_mask=[False] * 10,
+    get_birdview=True,
 )
 frames = []
 pbar = tqdm(range(50))
 for i in pbar:
     simulation.step(current_ego_agent_states=[])
-    collision, offroad, wrong_way = simulation.infractions
+    collision, offroad, wrong_way = simulation.infraction_rates
     pbar.set_description(
-        f"Collision rate: {100*np.array(collision).mean():.2f}% | "
-        + f"Off-road rate: {100*np.array(offroad).mean():.2f}% | "
-        + f"Wrong-way rate: {100*np.array(wrong_way).mean():.2f}%"
+        f"Collision rate: {100*collision:.2f}% | "
+        + f"Off-road rate: {100*offroad:.2f}% | "
+        + f"Wrong-way rate: {100*wrong_way:.2f}%"
```

Comment:
Instead of percentages, would it make more sense to print infraction rates as fractions? Something like `2/11`. It's easier to read and reconcile with the image than the percentages.